### PR TITLE
second and third section completed

### PR DESCRIPTION
**Pull request by eerapheal**
my first microverse project and finished mobile version of Ekepnisi portfolio.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -38,6 +38,193 @@ <h2 class="sub-tittle">I’am a Software Developer</h2>
...
       </section>
+      <!--  -->
+<section class="second-sec">
+  <h3 class="second-tittle">
+    My Recent Works
+  </h3>
+  <img width="327" height="237" src="./images/Img-Placeholder.png" alt="">
+  <h3 class="second-subtittle">
+    Multi-Post Stories
+  </h3>
```
Comment (CollinsTatang) [inline index.html]: ### **Great implementation of the project requirements**

- [ ] Please your design misses the horizontal line under the Recent Work title and your font doesn't match the [figmal design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27) font.
### **Take Look at Figmal Design**
figmalRecentWork
### **Your Design**
yourfont

File: index.html [hunk 2]
```diff
@@ -38,6 +38,193 @@ <h2 class="sub-tittle">I’am a Software Developer</h2>
...
+    A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+  </p>
+  <div>
+    <ul class="list-btn">
+      <li class="css-btn">css</li>
+      <li class="html-btn">html</li>
+      <li class="boot-btn">bootsrap</li>
+      <li class="ruby-btn">ruby</li>
+    </ul>
+    <button class="btn-top" type="button">See projests</button>
```
Comment (CollinsTatang) [inline index.html:61]: - [ ] Please fix your see project button and Multi-Post font to match the [figmal design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27)

### **Figmal design**
seeProButt
### **Your Design**
client

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @eerapheal, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required changes 
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips that can improve your code quality and help you get a better job.
_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -38,6 +38,194 @@ <h2 class="sub-tittle">I’am a Software Developer</h2>
...
+          <p class="pro-p">A daily selection of privately personalized reads; no accounts or sign-ups required. has
+          been the industry's standard</p>
+          <ul class="pro-ul">
+            <li class="pro-html">html</li>
+            <li class="pro-boot">bootsrap</li>
+            <li class="pro-rubby">Ruby</li>
+          </ul>
+        </div>
+        <button class="pro-btn" type="button">See Projests</button> 
+      </div>
```
Comment (franclobo) [inline index.html:88]: - [ ] **[OPTIONAL]** There are three ✌🏽 more cards than in the Figma template, please delete them to stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

image

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @eerapheal , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

- No linter errors ✔️ 
- Correct Github flow ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

Consider this suggestions:

- [ ] We must follow the [READMe template](https://github.com/microverseinc/readme-template/blob/master/README.md) to have a professional file. 💯 
- [ ] We could add a descriptive title on Pull Request. 😎 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 4]
```diff
@@ -105,3 +105,413 @@ body {
...
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.third-sec {
+  width: 375px;
```
Comment (ElsonOtake) [inline style.css:335]: - [OPTIONAL] Please avoid the use of a limited width size. Don't declare width or use a width of 100% and control the size using margins or paddings. It creates a more responsible page, a better user experience, and works in any kind of display. As not required by the previous reviewers, I consider that optional.
Check how your app could be in this case:

Current app
Screen Shot 2022-08-20 at 9 32 44 AM

Without width limitation:
Screen Shot 2022-08-20 at 9 21 57 AM
Screen Shot 2022-08-20 at 9 22 16 AM

File: style.css [hunk 5]
```diff
@@ -105,3 +105,413 @@ body {
...
+  background-color: #ff6b00;
+  color: #fff;
+  transition-duration: 2s;
+}
+
+.pro-btn:hover {
+  color: #ff6b00;
+  background-color: #fff;
+  transition-duration: 2s;
+}
```
Comment (ElsonOtake) [inline style.css:505]: - [OPTIONAL] Please create a border for the button when hovering following the Figma template. Check the other buttons for the same issue. As not requested by the previous reviewers, I consider that optional.

File: style.css [hunk 6]
```diff
@@ -105,3 +105,413 @@ body {
...
+.css-btn {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  width: 49px;
+  height: 40px;
+  background-color: #fff;
+  border: 1px solid #8993a4;
+  font-family: 'Inter', serif;
```
Comment (ElsonOtake) [inline style.css:172]: - [OPTIONAL] Please set up the font following the [Google fonts](https://fonts.google.com/) instruction using `<link>` or `@import`. Example: 
```
<link rel="preconnect" href="https://fonts.googleapis.com">
<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
<link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500&family=Roboto:wght@100;300;400;500;700&display=swap" rel="stylesheet">
```
As not requested by the previous reviewer. I consider that optional. Check other fonts for the same issue.

**Review by ElsonOtake — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi @eerapheal, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 

Minions

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️

## Optional suggestions

- [OPTIONAL] Please check the suggested changes in your code for improvements.
- [OPTIONAL] Please create a more professional Pull Request following the [Document your repo in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/clean_repo.md).

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. 👍 👍 

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by eerapheal** [comment]
thanks for the review

**Pull request merged**